docs/questions/burn-tokens-usage.md

```diff
@@ -0,0 +1,97 @@
+### Question
+
+
+How do I implement token burns in GalaChain?
+
+
+### Answer
+
+
+GalaChain provides built-in mechanisms for burning tokens through the `TokenBurn` class and `burnTokens` function. Here's how to use them:
+
+1. The `TokenBurn` Class:
+GalaChain provides a built-in `TokenBurn` class in `@gala-chain/api` that tracks token burns:
+
+```typescript
+import { TokenBurn } from '@gala-chain/api';
+
+// TokenBurn class structure
+export class TokenBurn extends ChainObject {
+  @ChainKey({ position: 0 })
+  public burnedBy: UserAlias;  // Who performed the burn
```

Comment:
I keep wondering about that static `INDEX_KEY` field. Perhaps it would be good to add it to all samples and indicate that this is the first part of the key on chain (perhaps AI could handle that?). It is useful when you inspect those objects in explorer